src/server/services/procedures/fbi-crime-data/types.js

```diff
@@ -219,7 +219,7 @@ const US_TERRITORIES = {
     'Montana': 'MT',
     'Nebraska': 'NE',
     'North Carolina': 'NC',
-    'North Dakota': 'ND',
+    'North Dakota': 'N'+'D', // split to pass spell check
```

Comment:
Is there another way to get around the spell checker?

Reply:
Sadly, looks like the finest-grain alternative is allowing `ad` in all `*.js` files or disabling spell check for this (entire) file specifically... Fortunately, this kind of hack wasn't needed in many places.

Reply:
Gotcha. Too bad we can't have skip words for individual files (or commands to skip lines). It should be fine. Overall, this is an improvement as it found a lot more typos than it added workarounds :)

Reply:
Yeah, there are some finer controls we can implement using it, though. Like the tool has a CLI that checks individual files, so we could probably just write a little python script that does file-specific ignores like this, maybe with an optional `_typos.toml` in each directory that's just a `map<file, ignores[]>`

Reply:
Side question: did you notice the crazy typos in the fbi crime data service? Crazy stuff like `vehcile` instead of `vehicle` and `liqour` instead of `liquor`... I checked, and apparently they really did misspell it in their API (correct spelling is a 404)... wild.